src/apprt/gtk-ng/class/surface.zig

```diff
@@ -560,13 +547,7 @@ pub const Surface = extern struct {
     }
 
     pub fn toggleCommandPalette(self: *Self) bool {
-        signals.@"toggle-command-palette".impl.emit(
-            self,
-            null,
-            .{},
-            null,
-        );
-        return true;
+        return self.as(gtk.Widget).activateAction("win.toggle-command-palette", null) != 0;
```

Comment:
Add a comment here that in the future we'll pass the target surface as a param. No need yet.